### PR TITLE
Common-ize some daemon methods

### PR DESCRIPTION
**Pull request by z3ntu**
@terrycain What's your opinion on something like this?

In the end something remotely similar to https://github.com/cyanogen/uchroma/blob/master/uchroma/server/data/keyboard.yaml#L100 is my "goal" (so declaring "features" on devices and not all the methods seperately).

## TODO
- [x] Fix the bugs 😉

### CODE REVIEW

File: daemon/openrazer_daemon/hardware/device_base.py [hunk 1]
```diff
@@ -23,7 +23,9 @@ class RazerDevice(DBusService):
     """
     BUS_PATH = 'org.razer'
     OBJECT_PATH = '/org/razer/device/'
+    # device methods available in all devices
```
Comment (z3ntu) [inline daemon/openrazer_daemon/hardware/device_base.py]: that comment line should be below the `METHODS = []` line

**Comment by terricain** [comment]
Looks ok, as long as it works :smile:

**Review by terricain — APPROVED** [review]

**Comment by z3ntu** [comment]
Doesn't work yet :/

**Comment by z3ntu** [comment]
I think this is ready to merge now.

**Pull request merged**